ygot/render.go

```diff
@@ -884,6 +884,27 @@ func ConstructInternalJSON(s GoStruct) (map[string]interface{}, error) {
 	})
 }
 
+// EmitRFC7951 renders the supplied interface to RFC7951-compatible JSON. The argument
+// supplied must be a valid type within a generated ygot GoStruct - but can be a member
```

Comment:
```suggestion
// supplied must be a valid type within a generated ygot GoStruct (i.e. all types except the root GoStruct type) - but can be a member
```

Reply:
Can you explain what you meant here please? I'm not sure I understood what `root` means in this context?

Reply:
I just meant that this cannot be used for the root of all the models, because from my reading of the code it seems that should be handled by `EmitJSON`.